easygems/healpix/__init__.py

```diff
@@ -17,7 +17,7 @@ def get_npix(dx):
     return healpy.nside2npix(dx.crs.healpix_nside)
 
 
-def isel_extent(dx, extent):
+def get_mask(dx, extent):
```

Comment:
Maybe this function should get a more specific name (e.g. `get_mask_for_extent` or `extent_to_mask`), to avoid possible later name clashes.

Reply:
I totally agree. I also wasn't happy with the name but couldn't find a good compromise of brevity and clarity

Reply:
I opted for `get_extent_mask`. If you are fine with it, I will rebase and merge

Reply:
👍